c_src/exec.hpp

```diff
@@ -213,31 +213,29 @@ struct CmdOptions {
     }
 
 public:
-    CmdOptions(int def_user=INT_MAX)
-        : m_tmp(0, 256), m_shell(true), m_pty(false)
-        , m_kill_timeout(KILL_TIMEOUT_SEC)
-        , m_kill_group(false)
+    explicit CmdOptions(int def_user=INT_MAX)
+        : m_tmp(0, 256)
         , m_is_kill_cmd(false)
-        , m_cenv(NULL), m_env_clear(false)
         , m_nice(INT_MAX)
         , m_group(INT_MAX), m_user(def_user)
-        , m_success_exit_code(0)
     {
         init_streams();
     }
-    CmdOptions(const CmdArgsList& cmd, const char* cd = NULL, const MapEnv& env = MapEnv(),
-               int  user = INT_MAX, int nice = INT_MAX, int group = INT_MAX,
-               bool is_kill_cmd=false)
-        : m_shell(true), m_pty(false), m_cmd(cmd), m_cd(cd ? cd : "")
-        , m_kill_timeout(KILL_TIMEOUT_SEC)
-        , m_kill_group(false)
+    CmdOptions(const CmdArgsList& cmd, const char* cd, const MapEnv& env,
+               int user, int nice, int group, bool is_kill_cmd)
+        : m_cmd(cmd), m_cd(cd ? cd : "")
         , m_is_kill_cmd(is_kill_cmd)
         , m_env(env)
-        , m_cenv(NULL),   m_nice(INT_MAX)
+        , m_nice(nice)
```

Comment:
This was also one compiler warning, the parameter nice was not used. Since this constructor was always called with nice = INT_MAX, this had no effect.

Reply:
nice ;)